README.md

```diff
@@ -57,7 +57,7 @@ dataset_cmd="--data_root $DATA_ROOT \
   --video_column $VIDEO_COLUMN \
   --caption_column $CAPTION_COLUMN \
   --id_token BW_STYLE \
-  --video_resolution_buckets 17x512x768 49x512x768 61x512x768 129x512x768 \
+  --video_resolution_buckets 49x512x768 \
```

Comment:
Why is this getting changed?

Reply:
This was incorrect when I merged LTX. I copied values from my multiresolution run but validation prompts and other settings from single resolution run